smartapps/smartthings/energy-alerts.src/energy-alerts.groovy

```diff
@@ -64,7 +64,7 @@ def meterHandler(evt) {
     def lastValue = atomicState.lastValue as double
     atomicState.lastValue = meterValue
 
-    def dUnit ? evt.unit : "Watts"
+    def dUnit = evt.unit ?: "Watts"
```

Comment:
The code doesn't compile, so it would have to be merged before the compile checking. Why do you want to separate it?